subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/AggregatingIncrementalAnnotationProcessingIntegrationTest.groovy

```diff
@@ -392,7 +390,7 @@ class AggregatingIncrementalAnnotationProcessingIntegrationTest extends Abstract
         succeeds "compileJava"
 
         then:
-        with(operations[':compileJava'].result.annotationProcessorDetails as List<CompileJavaBuildOperationType.Result.AnnotationProcessorDetails>) {
```

Comment:
This type wasn't actually true - The List contains JSON objects that have the same structure, but different types than the real build operations. I've adjusted it to make that clear and get rid of a compiler warning telling me that we were comparing incompatible types (String vs. Enum)